docs/docs/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Setting Up and Running Chronos
+---
+
+# Setting up and Running Chronos
+
+## Quickstart
+
+The [playa-mesos](https://github.com/mesosphere/playa-mesos) provides an easy way to try Mesos, Marathon and Chronos within a virtual machine using Vagrant.
+
+
+## Requirements
+
+These requirements are just to run Chronos. You will need additional packages to build Chronos from source (see the [Building from Source section](#build-from-source) below).
+
+* [Apache Mesos][Mesos] 0.20.0+
+* [Apache ZooKeeper][ZooKeeper]
+* JDK 1.6+
+
+
+## Install from Packages
+
+Mesosphere provides builds for Mesos and Chronos for major Linux distributions and OS X on their [downloads page](http://mesosphere.com/downloads/).
+
+## Install from Tarball
+
+Use the latest tagged Chronos release with Mesos 0.20+ as follows:
+
+```sh
+curl -0 https://github.com/mesos/chronos/archive/2.3.3.tar.gz
+tar xvf 2.3.3.tar.gz
+```
+
+<hr />
+
+## <a name="build-from-source"></a>Building from Source
+
+Follow these steps to install Chronos on Amazon Linux:
+
+### Install Dependencies
+
+#### Debian Linux:
+    sudo apt-get install autoconf make gcc cpp patch python-dev git libtool default-jdk default-jdk-builddep default-jre gzip libghc-zlib-dev libcurl4-openssl-dev
+
+#### Fedora Linux:
+    sudo yum install autoconf make gcc gcc-c++ patch python-devel git libtool java-1.7.0-openjdk-devel zlib-devel libcurl-devel openssl-devel cyrus-sasl-devel
+
+Make sure you're using Java 7: `sudo alternatives --config java`
+
+### Build and Install Mesos
+
+    git clone https://github.com/apache/mesos.git
+    cd mesos/
+    git checkout 
+    export JAVA_HOME=/usr/lib/jvm/java-1.7.0-openjdk-1.7.0.45.x86_64/
+    ./bootstrap
+    ./configure --with-webui --with-included-zookeeper --disable-perftools --enable-frame-pointers
+    make
+    sudo make install
+
```

Comment:
Let's have the option here of only installing Chronos and not building it. Most users will want to simply install it using yum install, for example.